boards/same54-xpro/doc.txt

```diff
@@ -0,0 +1,103 @@
+/**
+@defgroup    boards_same54-xpro Atmel SAM E54 Xplained Pro
+@ingroup     boards
+@brief       Support for the Atmel SAM E54 Xplained Pro board.
+
+## Overview
+
+The `SAME54 Xplained Pro` is an evaluation board by Atmel featuring an
+ATSAME54P20A SoC. The SoC includes a SAME54 ARM Cortex-M4F microcontroller.
+For programming the MCU comes with 256Kb of RAM and 1024Kb of flash memory.
+
+The same54-xpro is available from various hardware vendors for ~70USD (as of
+2019 March).
+
+## Hardware
+
+![same54-xpro image](http://www.microchip.com/_ImagedCopy/ATSAME54-XPRO_angle.jpg)
+
+
+### MCU
+| MCU        | ATSAME54P20A      |
+|:------------- |:--------------------- |
+| Family | ARM Cortex-M4F    |
+| Vendor | Atmel |
+| RAM        | 256Kb |
+| Flash      | 1024Kb             |
+| Frequency  | up to 120MHz |
+| FPU        | yes                |
+| Timers | 8 (16-bit)    |
+| ADCs       | 2x 12-bit (20 channels)           |
+| UARTs      | max 8 (shared with SPI and I2C)               |
+| SPIs       | max 8 (see UART)                  |
+| I2Cs       | max 8 (see UART)              |
+| Vcc        | 1.62V - 3.63V         |
+| Datasheet  | [Datasheet](http://ww1.microchip.com/downloads/en/DeviceDoc/60001507C.pdf) |
+| Board Manual   | [Board Manual](http://ww1.microchip.com/downloads/en/DeviceDoc/70005321A.pdf)|
+
+### User Interface
+
+1 User button and 1 LED:
+
+| Device | PIN |
+|:------ |:--- |
+| LED0   | PC18 |
+| SW0 (button) | PB31 |
+
+
+## Implementation Status
+
+| Device | ID        | Supported | Comments  |
+|:------------- |:------------- |:------------- |:------------- |
+| MCU        | same54    | partly    | DFLL clock not implemented |
+| Low-level driver | GPIO    | yes       | |
+|        | ADC       | yes   | |
+|        | PWM       | yes   | |
+|        | UART      | yes       | |
+|        | I2C       | unknown       | |
+|        | SPI       | yes       |  |
+|        | USB       | no        | |
+|        | RTT       | yes       | |
+|        | RTC       | yes       |  |
+|        | RNG       | unknown       |  |
+|        | Timer     | yes           | |
+|        | Ethernet  | no           | |
+
+
+
+## Flashing the device
+
+Connect the device to your Micro-USB cable using the port labeled as *DEBUG
+USB*.
+
+The standard method for flashing RIOT to the same54-xpro is using OpenOCD.
```

Comment:
I'm not against using openOCD but EDBG is usually faster to flash. Could you also add it ?
Selection should be done by the user using PROGRAMMER=edbg or PROGRAMMER=openocd in a Makefile.

Reply:
`sam0.inc.mk` defaults to EDBG anyway, so this section is wrong.